src/themes/dark.scss

```diff
@@ -8,11 +8,16 @@
   --wxc-banner-background: #{$wxc-gradation-dark-transparent};
   --wxc-banner-text-color: #{$wxc-white-alpha-95};
 
+  --wxc-icon--background-color: #{$wxc-black-alpha-100};
   --wxc-icon--color: #{$wxc-white-alpha-100};
   --wxc-icon--active-color: #{$wxc-green-50};
   --wxc-icon--muted-color: #{$wxc-red-50};
   --wxc-icon--error--gradient-start-color: #{$wxc-pink-50};
   --wxc-icon--error--gradient-stop-color: #{$wxc-red-60};
+  --wxc-icon--avatar-active-color: #{$wxc-green-40};
```

Comment:
let's keep the status colors separated from the icons, and define generic variables that only relate to status:
```scss
--wxc-person-status--active--color
--wxc-person-status--dnd--color
...
```

Reply:
they should go at the top with the other generic variables (text-color, video-background)